13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+We think this consideration is important - particularly when it comes to data-related issues that we consider to be objective, such as how variables are assumed and constructed to be dichotomous (such as variables for individual's gender) or categorical (such as variables for individual's race), when the truth may be that such variables are based on decisions that analysts or those collecting data made - decisions that may benefit from questioning. This consideration is also important when it comes to what data is used for teaching and learning. If names of individual's in data exclusively from individuals from majority racial or ethnic group, for example, some learners may implicitly perceive the content being taught to be designed for others. While we may think that such issues are better left to those we are teaching to decide later on, setting the stage in classes, courses, and other contexts in which data science is taught and learned can set an important precedent for the data our learners use and how they use it.
+
+### Provide a home base for learners to access resources (and to learn more)
+
+As we discuss in the next section, along with other important factors (such as learners' motivation and having a supportive atmospher), learning strategies can make a difference for learners. Especially when it comes to learning to do data science, there are many tools and resources to keep track of, such as:
+
+- How to download and install R
+- How to download and install R Studio
+- How to install packages
+- How to access resources related to the workshop or course (or simply other resources you wish to share)
+- How to contact the instructor 
+- How to get help and learn more
+
+Having a "home base," where you can remind learners to first look to for resources, can help to lower some of learners' demands in terms of remembering how these tools and resources can be accessed. One way to do this is through a personal website. Another is through GitHub pages. For some organizations, a proprietary learning management system - such as Desire2Learn, Blackboard, Moodle, or Canvas - can be helpful (especially if your learners are accustomed to using them). 
+
+### Draw on other resources
+
+In this section of this chapter, we presented some strategies for teaching data science. There are others that go more into depth on this topic from different perspectives, such as the following: 
+
+- [GAISE Guidelines](https://www.amstat.org/asa/education/Guidelines-for-Assessment-and-Instruction-in-Statistics-Education-Reports.aspx): guidelines for teaching statistics
+- [Teaching Tech Together](http://teachtogether.tech/#the-rules): a book on teaching programming
+- [Data Science for Undergraduates](https://www.nap.edu/catalog/25104/data-science-for-undergraduates-opportunities-and-options): a report on undergraduate data science education
```

Comment:
Possible addition is the new R Studio education site?

Reply:
a *great* idea

Reply:
added